<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniZinc IDE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabsClosable">
        <bool>true</bool>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="configuration">
        <attribute name="title">
         <string>Configuration</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Translation</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="checkBox">
                <property name="text">
                 <string>Data file</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_2"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>Browse</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>MiniZinc include path</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit"/>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QCheckBox" name="checkBox_2">
                <property name="text">
                 <string>Additional parameters</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_3"/>
              </item>
             </layout>
            </item>
            <item row="3" column="0" alignment="Qt::AlignLeft">
             <widget class="QCheckBox" name="checkBox_4">
              <property name="text">
               <string>Verbose</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QCheckBox" name="checkBox_3">
              <property name="text">
               <string>Optimize</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Solving</string>
           </property>
           <widget class="QCheckBox" name="checkBox_5">
            <property name="geometry">
             <rect>
              <x>490</x>
              <y>30</y>
              <width>131</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Print all solutions</string>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_6">
            <property name="geometry">
             <rect>
              <x>650</x>
              <y>30</y>
              <width>87</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Statistics</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>60</y>
              <width>62</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>#Threads</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="spinBox_2">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>60</y>
              <width>49</width>
              <height>24</height>
             </rect>
            </property>
           </widget>
           <widget class="QCheckBox" name="checkBox_7">
            <property name="geometry">
             <rect>
              <x>290</x>
              <y>60</y>
              <width>111</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Random seed</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="geometry">
             <rect>
              <x>420</x>
              <y>60</y>
              <width>113</width>
              <height>21</height>
             </rect>
            </property>
           </widget>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>30</y>
              <width>155</width>
              <height>26</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Solver</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>301</x>
              <y>30</y>
              <width>126</width>
              <height>26</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>#Solutions</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTextBrowser" name="outputConsole"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionGo_to_line"/>
    <addaction name="actionFind_Replace"/>
   </widget>
   <widget class="QMenu" name="menuMiniZinc">
    <property name="title">
     <string>MiniZinc</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionStop"/>
    <addaction name="actionCompile"/>
    <addaction name="separator"/>
    <addaction name="actionConstraint_Graph"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionNew_2"/>
    <addaction name="actionOpen_2"/>
    <addaction name="actionSave_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuMiniZinc"/>
   <addaction name="menuProject"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionGo_to_line">
   <property name="text">
    <string>Go to line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionFind_Replace">
   <property name="text">
    <string>Find/Replace</string>
   </property>
  </action>
  <action name="actionConstraint_Graph">
   <property name="text">
    <string>Constraint Graph</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Meta+C</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="text">
    <string>Compile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
